src/app/app-components/courses/courses-list/courses-list.component.ts

```diff
@@ -1,13 +1,20 @@
-import { Component, OnInit } from '@angular/core';
-import { Course } from 'src/app/app-models/course';
+import { Component, OnInit, Input } from '@angular/core';
+import { Course } from '@app/app-models';
+import { todayWithoutTime, yesterdayWithoutTime, tomorrowWithoutTime } from '@app/app-utils/utils';
```

Comment:
Just a small comment. Seems you could've created an `withoutTime` function which you would chain after the call for either `today`, `yesterday` or `tomorrow`. We don't need to make this change, OK?